coderd/database/dbmem/dbmem.go

```diff
@@ -724,17 +724,40 @@ func (q *FakeQuerier) getOrganizationMemberNoLock(orgID uuid.UUID) []database.Or
 }
 
 // getEveryoneGroupMembersNoLock fetches all the users in an organization.
-func (q *FakeQuerier) getEveryoneGroupMembersNoLock(orgID uuid.UUID) []database.User {
+func (q *FakeQuerier) getEveryoneGroupMembersNoLock(orgID uuid.UUID) []database.GroupMember {
```

Comment:
The `FakeQuerier` has a data field `groupMembers                  []database.GroupMember`. This should now be `groupMembers                  []database.GroupMemberTable`

Doing that will review some queries that need to be updated to include the joined information.